tests/e2e/performance/test_single_pvc_clone_perf.py

```diff
@@ -0,0 +1,172 @@
+"""
+Test to verify clone creation and deletion performance
+"""
+import logging
+import pytest
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework.testlib import (
+    skipif_ocs_version
+)
+from ocs_ci.framework.testlib import (
+    performance, E2ETest
+)
+from ocs_ci.ocs.resources import pvc
+from ocs_ci.ocs.resources import pod
+from tests import helpers
+
+
+logger = logging.getLogger(__name__)
+
+
+@skipif_ocs_version('<4.6')
+@pytest.mark.parametrize(
+    argnames=["interface_type", "pvc_size", "file_size"],
+    argvalues=[
+        pytest.param(
+            *[constants.CEPHBLOCKPOOL, '1', '600M'], marks=pytest.mark.polarion_id('OCS-2340')
```

Comment:
@ebenahar Yes, since there are 2 test cases in Polarion. Because the Ceph FS clones we have a bug on LSO  ( https://bugzilla.redhat.com/show_bug.cgi?id=1882504) while RBD clones are created properly.

Reply:
@ebenahar regarding max_num_of_clones - it's configureable in the code. Currently this is set to 1 ( since Karthick asked that we have a single pvc - single clone test). But of course in the future we will execute it for big ( 512) number of clones. so this parameter is a preparation for the future execution and measurements.

Reply:
we usually want each test iteration to update a different polarion id, so we will not have them overriding the results